ios/LocationChangeReceiver.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  LocationChangeReceiver.swift
+//  Magnet
+//
+//  Created by Francisco Jordano on 24/10/2016.
+//
+
+import Foundation
+import CoreLocation
+
+@objc(LocationChangeReceiver) class LocationChangeReceiver: NSObject, CLLocationManagerDelegate {
+  let locationManager:CLLocationManager = CLLocationManager()
+  var scanner: MagnetScanner! = nil
+  
+  @objc override init() {
+    super.init()
+    scanner = MagnetScanner(callback: self.onMagnetScanner)
+  }
+  
+  private func onMagnetScanner(result: Dictionary<String, AnyObject>) {
+    let url = result["url"] as! String
+    // Uncomment following line for debug
+    //NotificationsHelper.showNotification("Url we should notify \(url)")
```

Comment:
As we don't always notify (if we saw the url in the past 10 minutes), this line is helpful to see if we discovered something while we were in background.